src/js/components/TextInput/doc.js

```diff
@@ -1,5 +1,6 @@
 import { describe, PropTypes } from 'react-desc';
 
+import { plainPropType } from '../../utils/prop-types';
```

Comment:
The `plain="full"` is very specific for TextInput and not something we would like to encourage in other components (with the thought of keeping the API simple), hence there is no need to extract the logic to the utils file.

Reply:
Moved plainPropType to just be in TextInput